web-client/src/AppInstanceManager.tsx

```diff
@@ -18,13 +18,15 @@ export const AppInstanceManager = connect(
     appInstanceManagerHelper: state.appInstanceManagerHelper,
     confirmStayLoggedInSequence: sequences.confirmStayLoggedInSequence,
     resetIdleTimerSequence: sequences.resetIdleTimerSequence,
-    signOutSequence: sequences.signOutSequence,
+    signOutIdleSequence: sequences.signOutIdleSequence,
+    signOutUserInitiatedSequence: sequences.signOutUserInitiatedSequence,
```

Comment:
`signOutUserInitiatedSequence` is more or less just a wrapper for `signOutSequence` for consistency in nomenclature/conceptual boundaries given the new `signOutUserInitiatedSequence`. However, if we prefer to just have `signOutSequence` instead, I'm happy to make that change.

Reply:
Your instincts are correct here! I like that they are both explicit.